data/Australia/Senate/sources/instructions.json

```diff
@@ -71,6 +71,21 @@
         "from": "wikidata-raw",
         "source": "reconciliation/wikidata.csv"
       }
+    },
+    {
+      "file": "morph/twitter.csv",
+      "create": {
+        "from": "morph",
+        "scraper": "everypolitician-scrapers/twitter-australia-senate",
+        "query": "SELECT * FROM data"
```

Comment:
To avoid false positive diffs later, where the only change is the order that lines come out the database, it's good practice to always add an explicit `ORDER BY` here: e.g. https://github.com/everypolitician/everypolitician-data/blob/ad7e0b9d36826a387f957e14c4b3aa3aa5b202c4/data/Scotland/Parliament/sources/instructions.json#L32